proto/serverless/proto/common/v1/common.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
```

Comment:
I'm alright to remove the `proto` portion of the package. But I want to keep some folder structure. The node/typescript generators don't care about paths when they generate, but the others do. Go for instance needs the folder structure intact so that the generated module structure will be importable. 

So @medikoo can we agree to `serverless.common` and `serverless.instrumentation`?

Reply:
So, issue is that Go needs the folder structure that follows the _package_ name? And to be compatible we need `serverless` and `v1` folder here?

Reply:
Yup that's correct. I updated the PR that removes the `proto` portion so now it should be in a state that is good for both I think.